BO4E/COM/Rechenschritt.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Text.Json.Serialization;
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+using ProtoBuf;
+
+namespace BO4E.COM
+{
+    /// <summary>
+    /// Ein Berechnungsschritt ist Teil einer <see cref="BO4E.BO.Berechnungsformel"/>
+    /// </summary>
+    /// <remarks>Entspricht der SG9 in UTILTS</remarks>
+    public class Rechenschritt : COM
+    {
+        /// <summary>
+        /// Die BestandteilID des Rechenschritt (1-99999)
+        /// </summary>
+        /// <remarks>UTILTS SG8 RFF 1154 / SG</remarks>
+        [JsonProperty(Required = Required.Always, PropertyName = "rechenschrittBestandteilId")]
+        [JsonPropertyName("rechenschrittBestandteilId")]
+        public int RechenschrittBestandteilId { get; set; }
+
+        /// <summary>
+        /// Die ReferenzID des Rechenschritt (1-99999)
+        /// </summary>
+        /// <remarks>UTILTS SG8 RFF 1154 / SG</remarks>
+        [JsonProperty(Required = Required.Always, PropertyName = "referenzRechensschrittID")]
+        [JsonPropertyName("referenzRechensschrittID")]
+        public int ReferenzRechenschrittId { get; set; }
```

Comment:
Bei dem `PropertyName` und dem `JsonPropertyName` ist glaube ich ein `s` zu viel in `referenzRechenschrittID` oder?

Reply:
da7ac90be9b81d347ccb21d2cbe034040406db1a